### ISSUE TITLE
unable to start AirSonos

### ISSUE THREAD

**Issue by egiorno** [hassio-addons/addon-airsonos#8]
# Problem/Motivation

Hi all
I have the same issue described in issue "Unable to start AirSonos server" opened on june, 20th
Hi installed AirSonos from repository running on Hass io on ubuntu
The first time all worked fine. Then I deleted it and reinstalled it. 
From that moment the log, every second, shows

INFO: Starting the AirSonos server
[19:33:05.604702] main:1207 Starting airupnp version: v0.2.1.1 (Jun 22 2018 @ 19:33:41)
[19:33:05.606117] Start:955 UPnP init failed: -203

[19:33:05.606289] main:1248 Cannot start

the plugin is shown as active, but it doesn't actually start because of a missing component (upnp), I suppose

Any idea about how to investigate this issue? I'm new in docker environment...
where I can find upnp logs?

many thanks
egiorno

**Comment by frenck** [comment]
This might be caused by port 1900 being in use already.

Do you have any other add-on (or things on your Host OS) using this port already?
(e.g., the Plex add-on or the Unifi add-on).

**Comment by egiorno** [comment]
You are right! the fault was the unifi controller! now I tried to change the host port from 1900 to 1901 but, since I haven't any ubnt device (not yet) I cannot try if it works fine but I suppose autodiscovery will be broken...
unfortunately now airsonos finds all my upnp devices but sonos! during my test I've deleted from config file all references to my sonos device: maybe I've broken something?
Also trying to add sonos from HA integration a message appear saying that there are no sonos device in my network (obviously sonos is detected by my phone)
